plugins/modules/web_infrastructure/nomad_job.py

```diff
@@ -0,0 +1,250 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# (c) 2020, FERREIRA Christophe <christophe.ferreira@cnaf.fr>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: nomad_job
+author: "FERREIRA Christophe (@chris93111)"
+version_added: "1.2.0"
+short_description: Launch an Nomad Job.
+description:
+    - Launch an Nomad job. See
+      U(https://www.nomadproject.io/api-docs/jobs/) for an overview.
+options:
+    host:
+      description:
+        - FQDN of nomad server.
+      required: true
+      type: str
+    secure:
+      description:
+        - ssl connection.
```

Comment:
```suggestion
        - SSL connection.
```
SSL is also an abbrevation. (And an outdated one, SSL has been deprectaed for 9 years now. Better use TLS, or if you really want to mention SSL, use TLS/SSL.)

Also, a more complete sentence really helps users. Something like "Enables TLS/SSL to secure the connection."